frontend/src/components/Search/SearchForm/SearchForm.tsx

```diff
@@ -345,7 +355,47 @@ export const Search: React.FC<{
 				gap="6"
 				width="full"
 				color="weak"
+				position="relative"
 			>
+				<Box
+					position="absolute"
+					display="flex"
+					alignItems="center"
+					style={{
+						gap: 8,
+						left: 2,
+					}}
+				>
+					{queryTerms.map((term, index) => {
+						if (term.key === BODY_KEY && !term.value.length) {
+							return null
+						}
+
+						return (
+							<Box
+								key={index}
+								cssClass={styles.comboboxTag}
+								py="6"
+								position="relative"
+								whiteSpace="nowrap"
+								style={{
+									textOverflow: 'ellipsis',
+								}}
+							>
+								<IconSolidXCircle
+									className={styles.comboboxTagClose}
+									size={13}
+									onClick={() => {
+										handleItemRemove(index)
```

Comment:
Should add some tracking on this so we can easily view sessions using the new UI.